src/dataaggregator/aggregator/policyPoller_test.go

```diff
@@ -0,0 +1,96 @@
+package aggregator_test
+
+import (
+	"code.cloudfoundry.org/lager"
+	. "dataaggregator/aggregator"
+	"dataaggregator/aggregator/fakes"
+	. "dataaggregator/policy"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"time"
+)
+
+var _ = Describe("PolicyPoller", func() {
+	var (
+		database *fakes.FakeDB
+
+		poller     *PolicyPoller
+		testAppId1 = "testAppId"
+		policyStr1 = `
+		{
+		   "instance_min_count":1,
+		   "instance_max_count":5,
+		   "scaling_rules":[
+		      {
+		         "metric_type":"MemoryUsage",
+		         "stat_window_secs":300,
+		         "breach_duration_secs":300,
+		         "threshold":30,
+		         "operator":"<",
+		         "cool_down_secs":300,
+		         "adjustment":"-1"
+		      }
+		   ]
+		}`
+	)
+
+	BeforeEach(func() {
+		database = &fakes.FakeDB{}
+		logger := lager.NewLogger("PolicyPoller-test")
+		poller = NewPolicyPoller(logger, TestPolicyPollerInterval, database)
+	})
+	Context("Start", func() {
+		BeforeEach(func() {
+			poller.Start()
+		})
+		AfterEach(func() {
+			poller.Stop()
+		})
+		Context("when the poller is started", func() {
+			BeforeEach(func() {
+				database.RetrievePoliciesStub = func() ([]*PolicyJson, error) {
+					return []*PolicyJson{&PolicyJson{AppId: testAppId1, PolicyStr: policyStr1}}, nil
+				}
+			})
+			It("should retrieve policies for every interval", func() {
+				Eventually(database.RetrievePoliciesCallCount, 2500*time.Millisecond, 500*time.Millisecond).Should(Equal(2))
```

Comment:
Once you switch to use a Duration, this can be simplified to use the defaults values.